training_project/tests/test_unsuccessful_login.py

```diff
@@ -0,0 +1,10 @@
+from training_project.pages.login_page import LoginPage
+
+
+def test_unsuccessful_login(browser, login_page_load, invalid_user):
```

Comment:
You don't need browser and login_page_load at the same time. In other tests also you could think how to avoid using browser fixture in tests.